### PR TITLE
adal: use 0.4 to support federated account through wstrust 13 and 2005

### PR DESCRIPTION
**Pull request by yugangw-msft**

### CODE REVIEW

**Comment by yugangw-msft** [comment]
/CC:@derekbekoe
There is not that much to review, but FYI

File: src/command_modules/azure-cli-profile/setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 DEPENDENCIES = [
     'azure==2.0.0rc5',
-    'adal>=0.3.0'
+    'adal>=0.4.0'
```
Comment (mayurid) [inline src/command_modules/azure-cli-profile/setup.py]: Minor nit: Can we make this in alphabetical order for consistency?

Comment (yugangw-msft) [inline src/command_modules/azure-cli-profile/setup.py]: sure, will do

**Comment by derekbekoe** [comment]
:shipit:

**Pull request merged**